### DISCUSSION TITLE
rbenv | Lsp Solargraph setup please help

### DISCUSSION THREAD

**Discussion by BaritskiyIllya** [crivotz/nv-ide#10] [Q&A] [answered]
Hi, tnx fo sharing this great settings fo neovim (nv-ide)... great work !
I was desperate to find a solution to my problem, so I decided to write here, I will be unrealistically happy if you help me with advice
the problem is ====>  cmd [" solargrapf "] is not executable
i have installed gem solargraph and rbenv
and almost sure that the nv-ide settings for rvm
I know that I need to specify the path to the solargraph gem in the settings, but I cannot determine where in settings to do this

**Comment by crivotz** [comment, marked as answer]
```
require'lspconfig'.solargraph.setup{
  cmd = { "PATH/solargraph", "--stdio"},
  capabilities = require('cmp_nvim_lsp').update_capabilities(vim.lsp.protocol.make_client_capabilities())
}
```

PATH is where solargrpah is located ;)

**Reply by BaritskiyIllya** [reply]
Thank you very much for taking the time to help, now I understand where I made a mistake, it's not clear why, but for some reason I considered "PATH" as a global variable already initialized,
in my case everything worked
`cmd = {"/home/.../.rbenv/shims/solargraph", "--stdio"} `

**Reply by crivotz** [reply]
In my case with rvm is just as you wrote.
Solargraph is automatically in my path and doesn't need the path writing

**Reply by BaritskiyIllya** [reply]
anyway, thanks a lot, now I will continue to look for how to fix other errors))

**Reply by crivotz** [reply]
you are welcome ;)
